components/listingsPage/__tests__/Listings.test.js

```diff
@@ -0,0 +1,73 @@
+import { shallow } from "enzyme";
+
+import Listings from "../../../pages/listings";
+import Filters from "../filters";
+import Results from "../results"
+import Listing from "..";
+import Title from "../../title"
+
+
+import { findByTestAtrr} from "../../../Utils/test";
+
+ jest.mock("react-i18next", () => ({
+        withTranslation: () => (Component) => {
+          Component.defaultProps = { ...Component.defaultProps, t: () => [] };
+          return Component;
+        },
+      }));
+
+ 
+      describe("Listings Page", () => {
+        let listings;
+      
+        beforeEach(() => {
+          listings = shallow(<Listings />);
+        });
+      
+        afterEach(() => {
+          listings.unmount();
+        });
+      
```

Comment:
I think the discussion of  `beforeEach` vs `beforeAll` can also be applied here.